### PR TITLE
Add article on using systemd-analyze

### PR DESCRIPTION
**Pull request by sashko**
Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>

### CODE REVIEW

File: docs/articles/baseplatform/measuring-startup-performance-with-systemd-analyze.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+.. _measuring_startup_perf_with_systemd_analyze:
+
+Measuring startup performance with systemd-analyze
+==================================================
+
+`systemd` provides a tool called `systemd-analyze` that can be used to retreive statistics
```
Comment (JEderonn) [inline docs/articles/baseplatform/measuring-startup-performance-with-systemd-analyze.rst]: s/retreive/retrieve

**Review by JEderonn — COMMENTED** [review]
Minor typo, please fix before merge

File: docs/articles/baseplatform/measuring-startup-performance-with-systemd-analyze.rst (hunk 1 above)
Comment (sashko) [reply docs/articles/baseplatform/measuring-startup-performance-with-systemd-analyze.rst]: done

File: docs/articles/baseplatform/measuring-startup-performance-with-systemd-analyze.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+and other state and tracing information from the system and service manager, and then build
+an SVG plot showing units waiting for their dependencies.
+
+You can see which unit files are causing your boot process to slow down. You can then optimize
+your system accordingly.
+
+Using systemd-analyze
+---------------------
+
+To retreive statistics information from systemd, simply run:
```
Comment (thenor) [inline docs/articles/baseplatform/measuring-startup-performance-with-systemd-analyze.rst]:
```suggestion
To retrieve statistics information from systemd, run:
```

**Review by thenor — CHANGES_REQUESTED** [review]
Minor nit-picky comments.

File: docs/articles/baseplatform/measuring-startup-performance-with-systemd-analyze.rst (hunk 2 above)
Comment (sashko) [reply docs/articles/baseplatform/measuring-startup-performance-with-systemd-analyze.rst]: done

**Review by thenor — APPROVED** [review]

**Pull request merged**